src/backend/access/external/pxfutils.c

```diff
@@ -47,6 +52,24 @@ void port_to_str(char **port, int new_port)
 }
 
 /*
+ * get_hdfs_location_from_filespace
+ *
+ * Get hdfs location from pg_filespace_entry
+ * The returned path needs to be pfreed by the caller.
+ */
+void get_hdfs_location_from_filespace(char** path)
+{
+	Assert(NULL != path);
+	Oid dtsoid = get_database_dts(MyDatabaseId);
```

Comment:
It holds the OID of the current session's db. It is usually used to get the db's name or any other properties related to it, such as the filespace.
It is defined in globals.c and set in postinit.c's InitPostgres() which is called when a session is opened (see postgres.c's PostgresMain()).